packages/flutter/lib/src/material/text_field.dart

```diff
@@ -315,6 +372,7 @@ class TextField extends StatefulWidget {
     properties.add(IntProperty('maxLines', maxLines, defaultValue: 1));
     properties.add(IntProperty('maxLength', maxLength, defaultValue: null));
     properties.add(FlagProperty('maxLengthEnforced', value: maxLengthEnforced, ifTrue: 'max length enforced'));
+    properties.add(DiagnosticsProperty<GestureTapCallback>('onTap', onTap, defaultValue: false));
```

Comment:
I don't think we usually add callbacks to debugFillProperties, fwiw. Normally if we want to expose that we add something like the list of gestures that are listened for (they have short names) or some such, but in this case it's probably easier to just not mention it at all. Not a big deal though.